src/engine/Interfaces/ConstructorBags/AlgebraOptions.ts

```diff
@@ -0,0 +1,7 @@
+module ex {
+   export interface IVectorOptions {
```

Comment:
Agree with Erik, I would prefer to keep these side-by-side as they are directly related.